files/build_templates/rsyslog_plugin.conf.j2

```diff
@@ -7,8 +7,6 @@ template(name="prog_msg" type="list") {
     constant(value="\n")
 }
 
-$ModLoad omprog
```

Comment:
Why delete in this file, but 00-sonic.conf still have this line?

Reply:
omprog can only be loaded once in a module otherwise rsyslog complains. Since 00-sonic-conf will have this line, no need for other conf files to have. Since conf files get read in alphanumeric order, 00-sonic-conf will be read first so I added it there.